src/components/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -0,0 +1,16 @@
+import { useToggle } from "hooks/useToggle";
+import { useEffect } from "react";
+import style from "./ToggleSwitch.module.scss";
+
+type Props = {
+    isOn: boolean,
+    flip: () => void,
+}
+
+export function ToggleSwitch({ isOn, flip }: Props) {
```

Comment:
Put the useToggle here and accept a callback a a prop that is called when the toggle is switched. The callback should receive the isOn value